sys-block/sas3ircu/sas3ircu-16.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit mount-boot
+
+DESCRIPTION="LSI MPT-SAS3 controller management tool"
+HOMEPAGE="https://www.broadcom.com/products/storage/host-bus-adapters/sas-9300-8e#downloads"
+LICENSE="LSI"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~ppc64 ~x86 ~sparc-solaris ~x64-solaris ~x86-solaris"
+IUSE="efi doc"
+RESTRICT="strip fetch mirror"
+DEPEND=""
+RDEPEND=""
+QA_PREBUILT="opt/lsi/sas3ircu boot/efi/sas3ircu.efi"
+
+MY_PN=SAS3IRCU
+MY_P="${MY_PN}_P${PV}"
+
+DISTFILE_BIN=${MY_P}.zip
+DISTFILE_DOC=SAS3_IR_UG.pdf
+
+SRC_URI_BASE='https://docs.broadcom.com/docs-and-downloads/host-bus-adapters'
+SRC_URI="
+	${SRC_URI_BASE}/host-bus-adapters-common-files/sas_sata_12g_p${PV}/${DISTFILE_BIN}
+	doc? ( ${SRC_URI_BASE}/${DISTFILE_DOC} )"
+
+S="${WORKDIR}/${MY_P}"
+
+pkg_nofetch() {
+	elog "Broadcom has a mandatory click-through license on thier binaries."
+	elog "Please visit $HOMEPAGE and download ${DISTFILE_BIN} from the Mangement Software section."
+	elog "After downloading, move ${MY_P} into your DISTDIR directory"
+	if use doc; then
+		elog "Please also download 'SAS-3 Integrated RAID Configuration Utility User Guide' (${DISTFILE_DOC}) "
+		elog "and also place it into your DISTDIR directory"
+	fi
+	einfo $SRC_URI
+}
+
+supportedcards() {
+	elog "This binary supports should support ALL cards, including, but not"
+	elog "limited to the following series:"
+	elog ""
+	elog "LSI SAS 3004"
+	elog "LSI SAS 3008"
+}
+
+src_unpack() {
+	unpack ${DISTFILE_BIN}
+}
+
+src_install() {
+	exeinto /opt/lsi/
+	use amd64 || use x86 && doexe sas3ircu_rel/sas3ircu/sas3ircu_linux_x64_rel/sas3ircu
+	use ppc64 && doexe sas3ircu_rel/sas3ircu/sas3ircu_linux_ppc64_rel/sas3ircu
+	use amd64-fbsd && doexe sas3ircu_rel/sas3ircu/sas3ircu_freebsd_amd64_rel/sas3ircu
+	use x86-fbsd && doexe sas3ircu_rel/sas3ircu/sas3ircu_freebsd_i386_rel/sas3ircu
```

Comment:
gentoo/FreeBSD is dead, safe to remove those 2 lines above. keywords are already dropped.